thirdparty/zip/miniz.h

```diff
@@ -4401,7 +4401,7 @@ static MZ_FORCEINLINE mz_bool mz_zip_array_push_back(mz_zip_archive *pZip,
                                                      mz_zip_array *pArray,
                                                      const void *pElements,
                                                      size_t n) {
-  if (0 == n)
+  if (n == 0)
```

Comment:
`if (0 == n)` is better style when programming in C.
This change is not needed.